### PR TITLE
Update meteor-testapp to Meteor 1.8.2 / meteor-spk 0.5.0

### PR DESCRIPTION
**Pull request by kentonv**
Another update will be needed to work with Meteor 1.9, but first Sandstorm itself needs to be updated, which requires node-capnp to be updated for Node 12.

### CODE REVIEW

**Comment by ocdtrekkie** [comment]
This PR appears to exchange the "babel runtime" error documented in #3225 with:

> Error: Cannot find module "capnp"

File: meteor-testapp/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "start": "meteor run"
   },
   "dependencies": {
+    "@babel/runtime": "^7.8.4",
```
Comment (zenhack) [inline meteor-testapp/package.json]: Hm, should we have both `@babel/runtime` and `babel-runtime`? Not sure if there's actually a difference or if this is just a duplicate.

Comment (kentonv) [reply meteor-testapp/package.json]: IIRC if I tried removing either one, it broke... but I have no idea, feel free to play with it further.

**Comment by kentonv** [comment]
> This PR appears to exchange the "babel runtime" error documented in #3225 with:
>
> > Error: Cannot find module "capnp"

WTF it was totally working on my machine, but now I'm seeing that too.

**Comment by kentonv** [comment]
I fixed `meteor-spk` to put the `capnp` module where Meteor will find it, and also updated it to Meteor 1.10.1, and updated the testapp to use this version.

**Comment by xet7** [comment]
@kentonv 

Have you uploaded that fixed meteor-spk? I get capnp errors when starting Wekan that uses meteor-spk 0.5.1.

**Comment by kentonv** [comment]
@xet7 I just realized I hadn't pushed the git branch, did that now. But if you grabbed the 0.5.1 tarball, yes, that's the one that should work.

**Pull request merged**